source/guide_shaarli.rst

```diff
@@ -0,0 +1,96 @@
+.. highlight:: console
+
+.. author:: Kevin Jost <https://github.com/systemsemaphore>
+
+.. tag:: lang-php
+.. tag:: web
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/shaarli.png
+      :align: center
+
+#######
+Shaarli
+#######
+
+.. tag_list::
+
+Shaarli_ is a minimalist link sharing service. It can be used to share, comment and save interesting links. It is designed to be personal (single-user), fast and handy.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>`
+  * :manual:`Domains <web-domains>`
+
+License
+=======
+
+Shaarli_ is `Free Software <https://en.wikipedia.org/wiki/Free_software>`_. See `COPYING <https://github.com/shaarli/Shaarli/blob/master/COPYING>`_ for a detail of the contributors and licenses for each individual component.
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 7.4:
+
+::
+
+  [isabell@stardust ~]$ uberspace tools version show php
+  Using 'PHP' version: '7.4'
+  [isabell@stardust ~]$
+
+Set up your domain:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Clone the GitHub repository, install the 3rd-party PHP dependencies and copy the resulting shaarli directory to your document root:
```

Comment:
Is there a reason for using the "from source" method instead of "from release zip"? The latter one is much shorter.

https://shaarli.readthedocs.io/en/master/Installation#from-release-zip

Reply:
No particular reason for using the "from source" method.
For the release zip, the steps would be

cd to your document root:
`cd ~/html`
Get the latest release:
`wget https://github.com/shaarli/Shaarli/releases/download/v0.12.1/shaarli-v0.12.1-full.tar.gz`
Unpack to document root, not to a new "Shaarli" folder:
`tar xvzf shaarli-v0.12.1-full.tar.gz --strip-components=1`

I have successfully tested this just now.

Reply:
that seems much cleaner! can you adapt the guide, please?